gradle/documentation/changes-to-html.gradle

```
@@ -76,8 +76,16 @@ class ChangesToHtmlTask extends DefaultTask {
 
   def toHtml(File versionsFile) {
     def output = new ByteArrayOutputStream()
+    
+    // Check if the perl executable exists
+    def perlExecutable = project.externalTool("perl")
+    if (!new File(perlExecutable).exists()) {
```

I believe it can be simplified a bit:
```suggestion
    if (!file(perlExecutable).exists()) {
```

---

i tried gpt as well and no joy!